samples/bluetooth/peripheral_ots/src/main.c

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <string.h>
+#include <errno.h>
+#include <sys/printk.h>
+#include <zephyr.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/gatt.h>
+
+#include <bluetooth/services/ots.h>
+
+#define DEVICE_NAME             CONFIG_BT_DEVICE_NAME
+#define DEVICE_NAME_LEN         (sizeof(DEVICE_NAME) - 1)
+
+#define BT_GATT_OTS_UNSPECIFIED_OBJ_TYPE 0x2ACA
```

Comment:
Sure, I will move it there

Reply:
Decided to open an independent PR with it because I need to fix one existing OTS UUID declaration:

https://github.com/zephyrproject-rtos/zephyr/pull/27830